src/controllers/factorycontroller.js

```diff
@@ -68,6 +68,18 @@ class FactoryController extends OrderController
         return false;
     }
 
+    increaseTruckSize(order)
+    {
+        GAME.model.config.orderCapacity+=GAME.model.config.increaseTruckSizeQuantity;
+        this._updateMoney(- GAME.model.config.increaseTruckSizeCost);
+    }
+
+    decreaseTruckSize(order)
+    {
+        GAME.model.config.orderCapacity-=GAME.model.config.decreaseTruckSizeQuantity;
+        this._updateMoney(- GAME.model.config.decreaseTruckSizeCost);
+    }
```

Comment:
This can be a single function, or even better, handle this in the event. I think you may even leave the `descreaseTruckSize` out of the implementation.